mu4e/mu4e-view.el

```diff
@@ -397,6 +399,46 @@ article-mode."
       (set-buffer-modified-p nil)
       (read-only-mode))))
 
+(defun mu4e~view-gnus-display-mime (msg)
+  "Same as `gnus-display-mime' but add a mu4e headers to the message."
+  (lambda (&optional ihandles)
+    (gnus-display-mime ihandles)
+    (unless ihandles
+      (save-restriction
+        (article-goto-body)
+        (forward-line -1)
+        (narrow-to-region (point) (point))
+        (dolist (field mu4e-view-fields)
+          (let ((fieldval (mu4e-message-field msg field)))
+            (cl-case field
+              ((:path :maildir :user-agent :mailing-list :message-id)
+               (mu4e~view-gnus-insert-header field fieldval))
+              ((:flags :tags)
```

Comment:
The code seems to iterate over these fields because the rest are handled by Gnus.

I realize that [custom headers](https://www.djcbsoftware.nl/code/mu/mu4e/HV-Custom-headers.html) aren't displayed.

Could it be that the code need to consider that list union the list of custom headers defined in _mu4e-header-info-custom_?.

Reply:
Please test with the new commit.